ffi/src/sspi/sec_handle.rs

```diff
@@ -1494,4 +1502,228 @@ mod tests {
         let status = unsafe { DeleteSecurityContext(&mut new_sec_context) };
         assert_eq!(status, 0);
     }
+
+    #[test]
+    fn acquire_credentials_handle_w_null_credentials() {
+        // This test simulates initialize security context function call. It's better to run it using Miri
+        // https://github.com/rust-lang/miri
+        // cargo +nightly miri test
```

Comment:
suggestion: You can document tests with doc comments above the function
```suggestion
    /// This test simulates initialize security context function call. It's better to run it using Miri
    /// https://github.com/rust-lang/miri
    /// cargo +nightly miri test
    #[test]
    fn acquire_credentials_handle_w_null_credentials() {
```

Reply:
done